.github/workflows/macos_workflow.yml

```diff
@@ -43,3 +43,21 @@ jobs:
       with:
         # Repository upload token - get it from codecov.io. Required only for private repositories
         token: ${{secrets.CODECOV_TOKEN}}
+
+    - name: Gradle clean build
+      uses: eskatos/gradle-command-action@v1
+      with:
+        gradle-executable: "./test_runner/gradlew"
+        arguments: "-p test_runner clean build"
```

Comment:
I think that there is no point to run `check` and `build` together, if you need builded flank, just get rid of `check` step

Reply:
Thanks, changed